apisix/discovery/nacos/schema.lua

```diff
@@ -54,6 +54,49 @@ return {
         },
         access_key = {type = 'string', default = ''},
         secret_key = {type = 'string', default = ''},
+        hosts = {
```

Comment:
Maybe like https://github.com/bzp2010/apisix/blob/608a4c6041a0b2094a1cd1f2295be80ca900d316/apisix/discovery/kubernetes/schema.lua#L114-L216 would be a better way? Users have the option to continue using the single registry mode, or switch to the multi-registry mode altogether.

Reply:
If we change the configuration file to the above format, the redundant nesting called `hosts` is unnecessary, i.e. the configuration should be:

```
discovery:
  nacos:
    host: xxx
```

Or,

```
discovery:
  nacos:
    - id: default
      host: xxx
    - id: test
      host: xxx
```

Reply:
great!